src/shared/constants.ts

```diff
@@ -2,3 +2,57 @@ export const MOLECULAR_PROFILE_MUTATIONS_SUFFIX = '_mutations';
 export const MOLECULAR_PROFILE_UNCALLED_MUTATIONS_SUFFIX = `${MOLECULAR_PROFILE_MUTATIONS_SUFFIX}_uncalled`;
 export const MUTATION_STATUS_GERMLINE = 'Germline';
 export const PUTATIVE_DRIVER = 'Putative_Driver';
+
+export const SAMPLE_CANCER_TYPE_UNKNOWN = 'Unknown';
+
+/* cbioportal api request arguments */
+export const enum REQUEST_ARG_ENUM {
```

Comment:
these should already be protected by typing on the methods themselves.  i.e. you can't pass an incorrect value as an argument.